### ISSUE TITLE
moer-mpg-data-analysis_unit09

### ISSUE THREAD

**Issue by dirkzeuss** [GeoMOER/moer-mpg-data-analysis#9]

**Comment by GeoKL** [comment]
In Chapter "Time Series" I read this:

"Stationarity means that the time series has no trend and has similar variance over the entire time span. This is clearly not the case in the temperature data because of the strong seasonality regardless of any long term trend."

So, I don't understand, why strong seasonality is a reason to imply that our time series is not stationary. I thought, stationary means, that the "pattern" of changes stays the same, which is true for seasonality, isn't it? 

So, if we want to claim that air temperature is not a stationary time series, we would have to argue, that over years, it gets warmer/colder - so there is a trend. 
Or did  I get this wrong?

**Comment by dirkzeuss** [comment]
You are right, if you think of “seasonality” as the result of a deterministic process like for example a sine function, then this would not imply non-stationarity. 

However, stationarity in our case is an attribute of the non-deterministic part of the data distribution, that is the deviation from e.g. a (“perfect” = deterministic) wave function.

See also here: https://stats.stackexchange.com/questions/172979/is-a-model-with-a-sine-wave-time-series-stationary and here: https://towardsdatascience.com/stationarity-in-time-series-analysis-90c94f27322
